config/config.json

```diff
@@ -1,18 +1,13 @@
 {
-    "general": {
-    },
-    "features": {
-    },
-    "log": {
-    },
-    "email": {
-    },
-    "oauth": {
-    },
+    "general": {},
+    "features": {},
+    "log": {},
+    "email": {},
+    "oauth": {},
     "connections": [
         {
-            "type": "webdav",
-            "label": "WebDav"
+            "type": "ftps",
```

Comment:
I'm not a fan of enabling FTPs by default for everyone

Reply:
ok